frontends/concrete-python/README.md

```diff
@@ -19,10 +30,24 @@ cd ../../compilers/concrete-compiler/compiler
 make python-bindings
 
 # set bindings build directory as an environment variable
+# *** NOTE ***: You must use the Release build of the compiler! 
+# For now, the Debug build is not compatible with concrete-python
 export COMPILER_BUILD_DIRECTORY=$(pwd)/build
 echo "export COMPILER_BUILD_DIRECTORY=$(pwd)/build" >> ~/.bashrc
 
 # run tests
 cd ../../../frontends/concrete-python
 make pytest
 ```
+
+### VSCode setup
+
+Alternatively you can use VSCode to develop Concrete-Python:
+
+Suppose the compiler bindings were built in `../../compilers/concrete-compiler/compiler/build`:
+
+- Create a `.env` file in the concrete-python root directory
+- Add to it `PYTHONPATH=$(PYTHON_PATH):../../compilers/concrete-compiler/compiler/build/tools/concretelang/python_packages/concretelang_core/`
+- Add to it `LD_PRELOAD=../../compilers/concrete-compiler/compiler/build/lib/libConcretelangRuntime.so`
```

Comment:
I wouldn't embed a relative path to an environment variable, suggest them to use `$(pwd)` maybe?